### ISSUE TITLE
IncomingDataMessageBox doesn't remember checkbox state

### ISSUE THREAD

**Issue by che85** [SlicerProstate/SliceTracker#348]
This bug was introduced with commit https://github.com/SlicerProstate/SliceTracker/commit/e97265e2349d0de6e27f0bfd4342440f9f179081

Since we are working on the new Slicer version with the master branch, we should remove/replace 4.8 features entirely from that branch. 

We still have the 4.8 branch which is compatible with older Slicer versions.

@fedorov any thoughts?

**Comment by fedorov** [comment]
Yes, I agree, this makes complete sense to me. This should also be mentioned somewhere in the documentation and/or readme.
